### ISSUE TITLE
Need start meteor several times to build vendor/site files

### ISSUE THREAD

**Issue by valpet** [Semantic-Org/Semantic-UI-Meteor#90]
I have an issue when i have an empty custom.semantic.json file, as seen in the image below.

<img width="511" alt="empty file before starting meteor" src="https://cloud.githubusercontent.com/assets/596632/14231490/d26d1292-f985-11e5-8eb3-b7c3919ebd83.PNG">

When I start meteor it builds the app and I get the .custom.semantic json file, but no vendor files. At this stage it does not matter how many times I restart meteor, I won't get the vendor/site files.

<img width="494" alt="generated files but no vendor files after starting meteor once" src="https://cloud.githubusercontent.com/assets/596632/14231502/2cbe5436-f986-11e5-932e-b417158ccb6a.PNG">

If I delete the .custom.semantic.json file and restart meteor again, it will generate the vendor/site files for me.

<img width="502" alt="vendor files created after removing dot custom file and restarting meteor again" src="https://cloud.githubusercontent.com/assets/596632/14231506/5ad90e88-f986-11e5-90c4-0404d2da96c9.PNG">

While I know how to workaround this issue, it's still a really annoying issue. Especially when combined with another bug (that I will post an issue for as soon as it happens again so I can get screenshots) that forces me to delete all files but the custom.semantic.json and clear out all content of said file before a regeneration without errors is possible. This last piece is a side-problem that in combination with the problems described in this issue makes my workflow crippled.

**Comment by flemay** [comment]
Hi! Have you followed usage section from the Readme?

**Comment by valpet** [comment]
Yes, I have. All works fine until that second issue (ENOENT something while trying to load the custom.semantic.json file (I will post more details as soon as that one happens again, but it happens now and then and I have not yet figured out the cause). But to solve that issue, as meteor won't start, I have to clear out the generated files and start over with a clean custom.semantic.json file. That is when this issue, described in the original post, happens.

**Comment by flemay** [comment]
Thanks for replying. So this current issue is not really one right? If so we can close it?

**Comment by valpet** [comment]
This is in my opinion a separate issue that I encounter when working around the issue that forces me to wipe my files, as mentioned before.

**Comment by flemay** [comment]
The first issue you seem to experience is not an issue as it does not generate on purpose. Maybe changing the way it works would be better. :)

**Comment by valpet** [comment]
If I have a clean custom.semantic.json and no other files in my directory and when starting meteor it only generates a .custom.semantic.json file and fills both files with default data. No other files are generated; no site, no themes, no definitions. Is that not an issue? You can see in my screenshots how I reproduced the problem, step by step.

**Comment by flemay** [comment]
Yes it is expected to be like this. Point number 3 in the usage section. I wanted the developer to choose first instead of generating files straight away.

**Comment by valpet** [comment]
I did read the usage guide, but I guess I never understood how it works. Now when I better understand the **why**, I also better understand the **how**. May I suggest you add a little section of how it works, for other people as dense as me? :)

Thank you for your patience
